### DISCUSSION TITLE
ignoring encrypted files in .chezmoiignore

### DISCUSSION THREAD

**Discussion by Archie-2021** [twpayne/chezmoi#3685] [Q&A]
Hey , 

Yesterday I had a hard time with ignoring an encrypted file. I later found out that for ignoring it I have to add the suffix that create in Source Directory (in my case .asc) to the name of file. I was very confused because for ignoring we are supposed to use the name of the file in Target Directory which don't contain any such an extension.

Is this a bug of I am missing something in Encryption settings. Encryption works fine otherwise for me.

**Comment by twpayne** [comment]
I'm unable to reproduce this. Do you have a minimal example that demonstrates the problem?

**Comment by Archie-2021** [comment]
In my case I have a bookmarks files which I encrypted with gpg. I through an error on my work machine because I don't have my gpg key over there. 

Path to file on Target directory is:

.local/share/share-ari/bookmarks

So I should put this line in my .chezmoiignore file template (which only applies to my work machine). This line works:

.local/share/share-ari/bookmarks*

but not with this line:

.local/share/share-ari/bookmarks

**Reply by twpayne** [reply]
Thank you for the follow-up information. I don't think this is related to encryption.

Put the following in your `.chezmoiignore`:

```
.local/share/share-air/bookmarks/**
```

The background info is that the syntax of `.chezmoiignore` isn't quite the same as the syntax of `.gitignore` because git doesn't care about directories but chezmoi does. I should write an FAQ entry about this.
